MapboxNavigation/NavigationMapView.swift

```diff
@@ -194,8 +183,6 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
             if let userCourseView = userCourseView {
                 if let location = userLocationForCourseTracking {
                     updateCourseTracking(location: location, animated: false)
-                } else {
-                    userCourseView.center = userAnchorPoint
```

Comment:
> Remove the need to specify center for puck view - use the approach where content inset is keeping it centered.

We need to check whether undoing #2047 ends up regressing #1731 and #1949. Also, does the puck consistently remain at `userAnchorPoint` when simulation is sped up?

/ref @frederoni

Reply:
> Also, does the puck consistently remain at userAnchorPoint when simulation is sped up?

Yes, it remains. It is always centered to padding area and no need to reposition it. Attached video to show:
https://www.dropbox.com/s/wori62tfhp69feg/RPReplay_Final1565767435.mp4?dl=0

Also, the issue here is fixed: https://github.com/mapbox/mapbox-gl-native/issues/15233#issuecomment-521127342